src/components/atoms/statusDot/StatusDot.tsx

```diff
@@ -0,0 +1,27 @@
+import { Text } from "@ledgerhq/react-ui";
+import { ReactNode } from "react";
+import "./styles.css";
+
+type Props = {
+  children: ReactNode;
+  status: "success" | "loading" | "error";
+};
+
+export function StatusDot({ children, status }: Props) {
+  let textColor = "neutral.c50";
+
+  if (status === "success") {
+    textColor = "success.c50";
+  } else if (status === "error") {
+    textColor = "error.c50";
+  }
```

Comment:
I'd suggest creating a `colors` object outside the component using the status as keys so you can get the color with `color={colors[status]}`

Reply:
Done 👍 : https://github.com/LedgerHQ/wallet-connect-live-app/pull/168/commits/2ada0045225560358579c33cf00902c958070bb9